### DISCUSSION TITLE
Errors deploying the stack

### DISCUSSION THREAD

**Discussion by azarei** [cat-box/aws-foundry-ssl#29] [Q&A] [answered]
I've just been trying to run the stack but I've been getting errors in the deployment but not sure where to take it from here. I've tried it several times and it still won't go all the way through.

It errors out in the FoundryBucket step, claiming the 'foundry-games' already exists, even though I can't see that anywhere except in the key pair.  Any help or direction to deeper stack troubleshooting would be appreciated. Thank you.

![image](https://user-images.githubusercontent.com/1683547/124373235-c1fb5a80-dc4d-11eb-98ef-e6d0f90c1788.png)

**Comment by cat-box** [comment, marked as answer]
there are a number of possibilities... most likely one of the following two is what's happening.

1. you previously made a S3 bucket named "foundry-games" and it was not torn down. See [S3 Bucket](https://github.com/cat-box/aws-foundry-ssl/wiki/FAQ#s3-bucket) section in the FAQ.
2. as it suggests, the bucket name is already taken. S3 bucket names are globally unique and shared by ***all*** AWS accounts. "foundry-games" as a bucket name has already been taken by another user. You should choose another name.

If neither of these resolve your issue, send me a PM on reddit with your stack deployment parameters.

**Reply by azarei** [reply]
Thank you! That did it.
